tree/ntuple/v7/src/RNTupleModel.cxx

```diff
@@ -69,7 +68,7 @@ ROOT::Experimental::RNTupleModel::RProjectedFields::EnsureValidMapping(const Det
          // Source and target are children of the same collection
          return RResult<void>::Success();
       }
-      if ((fieldMap.count(targetBreakPoint) > 0) && fieldMap.at(targetBreakPoint) == sourceBreakPoint) {
+      if ((fieldMap.find(targetBreakPoint) != fieldMap.end()) && (fieldMap.at(targetBreakPoint) == sourceBreakPoint)) {
```

Comment:
I was actually thinking of reusing the iterator returned by `find()`, as in
```suggestion
      if (auto it = fieldMap.find(targetBreakPoint); it != fieldMap.end() && it->second == sourceBreakPoint) {
```

Reply:
[Done](https://github.com/root-project/root/pull/11973/commits/e9481edc2ac7d7363a9bd1755325900f4779c8c1)